s3-b2bi-s3/template2.yaml

```diff
@@ -0,0 +1,69 @@
+AWSTemplateFormatVersion: '2010-09-09' 
+Transform: AWS::Serverless-2016-10-31
+Description: SAM Template that creates an EC2 Instance, an Amazon S3 Express One Zone directory bucket and required IAM Role to access the bucket from the instance
```

Comment:
The description does not match the contents of the file

Reply:
Fixed.